packages/lib/src/dropdown/types.ts

```diff
@@ -76,6 +76,10 @@ type Props = {
    * Value of the tabindex attribute.
    */
   tabIndex?: number;
+  /**
+   * The title of the dropdown.
```

Comment:
I suggest to comment it as in other components. From the Button documentation:

![image](https://github.com/user-attachments/assets/bc0f54a2-58f7-4df2-b599-b4a5979df5cb)